### PR TITLE
plot_operator_breakdown: Remove relative share, fix warning

### LINKED ISSUE #2250: plot_operator_breakdown uses arithmetic mean on percentages
```
df.loc["Relative"] = df.head(-1).sum() / df.head(-1).count()
```

Should be the geomean instead? Not as clear cut as the compare_benchmarks case.

### PR DESCRIPTION
**Pull request by mrks**
fixes #2250

### CODE REVIEW

**Comment by Bouncner** [comment]
`ax.set_yticklabels(["{:,.0%}".format(x) for x in ax.get_yticks()])`

Since the removed line looks reasonable to me (it's percentages, right?): could you add a screenshot of the change?

File: scripts/plot_operator_breakdown.py [hunk 1]
```diff
@@ -117,7 +118,7 @@
         bar.set_linewidth(0)
 
 # Set labels
-ax.set_yticklabels(["{:,.0%}".format(x) for x in ax.get_yticks()])
```
Comment (Bouncner) [inline scripts/plot_operator_breakdown.py:120]: ax.set_yticklabels(["{:,.0%}".format(x) for x in ax.get_yticks()])

Comment (mrks) [reply scripts/plot_operator_breakdown.py:120]: ?

Comment (Bouncner) [reply scripts/plot_operator_breakdown.py:120]: Copy/paste fail ...

I wanted to ask for a screenshot as the removed line looks pretty good to me (it's percent and that's what the formatting does, right?).

Comment (mrks) [reply scripts/plot_operator_breakdown.py:120]: Does the same thing, but if the yticks ever change (which they don't), the formatting would be broken. That's why we were getting a warning there. Also, using the PercentFormatter is just the cleaner, canonical way of doing this.

**Comment by mrks** [comment]
Mmh, now that we have a better assertion in place, that spurious fail does not show up again. It's unrelated to this PR anyway.

**Review by Bouncner — APPROVED** [review]

**Pull request merged**